src/lightning/pytorch/_graveyard/xpu.py

```diff
@@ -0,0 +1,34 @@
+import sys
+from typing import Any
+
+import lightning.pytorch as pl
+
+
+def _patch_sys_modules() -> None:
+    self = sys.modules[__name__]
+    sys.modules["lightning.pytorch.accelerators.xpu"] = self
+
+
+class XPUAccelerator:
```

Comment:
I don't think it is needed as XPU newer was part of the codebase yet

Reply:
Oh, sure. Let me remove it.